app/src/main/java/com/andrew/tasky/presentation/agenda/AgendaViewModel.kt

```diff
@@ -32,19 +32,15 @@ class AgendaViewModel@Inject constructor(
 
     fun setDateSelected(dateUserSelected: LocalDate) {
         _dateSelected.value = dateUserSelected
-        _calendarDateItemList.value.map { item ->
-            item.isSelected = dateUserSelected == item.date
+        _calendarDateItemList.value = calendarDateItemList.value.map { item ->
+            CalendarDateItem(
+                dateUserSelected == item.date,
+                item.date
+            )
```

Comment:
I'd always stick to using named parameters if you can (if it's a Kotlin function). That makes your code a lot easier to scan. In your IDE, you might get these hints which tell you about the type of something, but in PRs I can't see what these values represent (here it's clear, but that's not always the case)